daemon/src/maker_inc_connections.rs

```diff
@@ -33,6 +33,12 @@ use xtras::AddressMap;
 use xtras::SendAsyncSafe;
 use xtras::SendInterval;
 
+static NUM_CONNECTIONS_GAUGE: conquer_once::Lazy<prometheus::IntGauge> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_int_gauge!("num_connections", "The number of active connections.")
```

Comment:
Proposal: rename to number of connected takers or similar. Number of connections might too ambiguous

Reply:
We can do that here but in the near future (once we have a network layer based on libp2p), maker and taker will run the same network actor so we can't easily distuingish the metric based on the role.

Is there a way in prometheus to take the source of a metric into account in its name?

Reply:
> We can do that here but in the near future (once we have a network layer based on libp2p), maker and taker will run the same network actor so we can't easily distuingish the metric based on the role.

That's fine, I meant the comby of `num` and `connections` is ambigious. 
Maybe call it p2p connection or incomming connections? When I hear `connections` it could mean various things: `http_connections`, `outgoing_connections`, ... I feel like we need to add one more thing to this field :D 

> Is there a way in prometheus to take the source (i.e. which HTTP endpoint was polled) of a metric into account in its name?

yes, each _scrape_ target has a name (http endpoint), which will end up as a label to this metric.

Reply:
Okay. What do you think of `num_p2p_connections`?

Reply:
> > Is there a way in prometheus to take the source (i.e. which HTTP endpoint was polled) of a metric into account in its name?
> 
> yes, each _scrape_ target has a name (http endpoint), which will end up as a label to this metric.

Nice, we can add `maker` and `taker `labels there then!

Reply:
I recorded an issue for this here: https://github.com/itchysats/cloud-conf/issues/18